spec.bs

```diff
@@ -185,6 +185,20 @@ dictionary AppHistoryReloadOptions : AppHistoryNavigationOptions {
 };
 </xmp>
 
+<div algorithm>
+  An {{AppHistory}} |appHistory| <dfn for="AppHistory">has entries and events disabled</dfn> if the following steps return true:
+
+  1. Let |browsingContext| be |appHistory|'s [=relevant global object=]'s [=Window/browsing context=].
+
+  1. If |browsingContext| is null, then return true.
```

Comment:
When is the browsing context null?

Reply:
Ah, is it when the document isn't active?

Reply:
Yeah, exactly.